lsp/mdx_analyzer.lua

```diff
@@ -0,0 +1,25 @@
+local function get_typescript_server_path(root_dir)
+  local project_root = vim.fs.dirname(vim.fs.find('node_modules', { path = root_dir, upward = true })[1])
+  return project_root and vim.fs.joinpath(project_root, 'node_modules', 'typescript', 'lib') or ''
+end
+
+---@brief
+---https://github.com/mdx-js/mdx-analyzer
+---
+---`mdx-analyzer`, a language server for MDX
+
+return {
+  cmd = { 'mdx-language-server', '--stdio' },
+  filetypes = { 'mdx' },
+  root_markers = { 'package.json' },
+  single_file_support = true,
```

Comment:
this is not a valid field in vim.lsp.config. should be `workspace_required = false` (requires Nvim 0.11.1+)